cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1500,6 +1500,36 @@ declare namespace Deno {
     request: Request,
   ): Promise<[Deno.Conn, Uint8Array]>;
 
+  /** **UNSTABLE**: New API, yet to be vetted.
+   *
+   * Allows "hijacking" the connection that the request is associated with. This
+   * can be used to implement protocols that build on top of HTTP (eg.
+   * {@linkcode WebSocket}).
+   *
+   * The returned promise returns the underlying connection.
+   *
+   * ```ts
```

Comment:
Let's not expose this API on the `Deno` namespace for now. The immediate goal is to wire it up in `ext/node` and I think we don't need to provide public API for now